fundingmanager.go

```diff
@@ -1221,7 +1233,13 @@ func (f *fundingManager) handleFundingOpen(fmsg *fundingOpenMsg) {
 	}
 
 	for _, c := range channels {
-		if c.IsPending {
+		// Pending channels that have a non-zero thaw height were also
+		// created through a canned funding shim. Those also don't
+		// count towards the DoS protection limit.
+		//
+		// TODO(guggero): Properly store the funding type (wallet, shim,
```

Comment:
to resolve this TODO, would we need db change?

Reply:
Yes. Though with TLV not necessarily a migration. Would like to get #4517 in first that introduces the TLV for channel serialization.